player/lua/osc.lua

```diff
@@ -1796,7 +1796,11 @@ function osc_init()
 
     ne.content = function ()
         if mp.get_property("pause") == "yes" then
-            return ("\238\132\129")
+            if mp.get_property("play-dir", "forward") ~= "backward" then 
```

Comment:
This should be updated to `play-direction` to not log a warning.

Reply:
Could also be shortened to:

```lua
return mp.get_property("play-direction") == "forward"
       and "\238\132\129"
       or "\238\132\144"
```

I have no idea why some other osc.lua functions put parentheses here but they don't do anything.